src/BaGet.Core.Server/Extensions/IRouteBuilderExtensions.cs

```diff
@@ -6,40 +7,25 @@ namespace BaGet.Extensions
 {
     public static class IRouteBuilderExtensions
     {
-        public static IRouteBuilder MapServiceIndexRoutes(this IRouteBuilder routes)
+        public static IEndpointRouteBuilder MapServiceIndexRoutes(this IEndpointRouteBuilder endpoints)
```

Comment:
Is it possible to port to endpoint routing in a separate pull request? I'd like to minimize the changes as the risk of breaking changes is already quite high.

Reply:
yes, i'll update the PR shortly